docs/source/index.rst

```diff
@@ -59,10 +59,10 @@ portability and maintainability of an application's source code.
 Installation
 ############
 
-The latest release of CBI is version 1.2.0. To download and install this
+The latest release of CBI is version 2.0.0. To download and install this
 release, run the following::
 
-    $ git clone --branch 1.2.0 https://github.com/intel/code-base-investigator.git
+    $ git clone --branch 2.0.0 https://github.com/intel/code-base-investigator.git
```

Comment:
Do you want to roll this into a single step for users?

```console
pip install git+https://github.com/intel/code-base-investigator@2.0.0
```

Unless you're developing CBI, I don't think this omits anything from the workflow?

Reply:
That's much better.  I've made this change in 827911f42b3a7d674d2de35178a8e8537965c495, for both the README and the docs.  I left it in expanded form for the example showing how to install it in a virtual environment.